### DISCUSSION TITLE
file upload

### DISCUSSION THREAD

**Discussion by ahmadali419** [konnco/filament-import#160] [Q&A]
i have updated the package to this version 
but sitll this issue is coming
File [/var/www/html/velux-rewards-adminportal-filament/resources/livewire-tmps/01J679Z8H1NZMTV0Y6F55Q9JD2.csv] (tmp-for-test) does not exist and can therefor not be imported.

**Comment by irwansaktip** [comment]
trying this https://github.com/konnco/filament-import/discussions/161#discussioncomment-10634476
